CenterForOpenScience/osf.io
# [ENG-2761] Make a default registration schema for each provider

## Purpose

Registration providers might want a default schema. This adds that capability

## Changes

Adding to the provider model, Creating default sorting on the api. Adding functionality to modify the default schema on the admin app.

## QA Notes

Please make verification statements inspired by your code and what your code touches.
- Verify
- Verify

What are the areas of risk? N/A

Any concerns/considerations/questions that development raised? N/A

## Documentation

N/A

## Side Effects

N/A
## Ticket

https://openscience.atlassian.net/browse/ENG-2761
